packages/hooks/src/useContentFlag.ts

```diff
@@ -0,0 +1,18 @@
+import { ContentFlag } from '@deriv/shared';
+import { useStore } from '@deriv/stores';
+
+const useContentFlag = () => {
+    const { traders_hub } = useStore();
+    const { content_flag } = traders_hub;
+
+    return {
+        low_risk_cr_non_eu: content_flag === ContentFlag.LOW_RISK_CR_NON_EU,
+        low_risk_cr_eu: content_flag === ContentFlag.LOW_RISK_CR_EU,
+        high_risk_cr: content_flag === ContentFlag.HIGH_RISK_CR,
+        cr_demo: content_flag === ContentFlag.CR_DEMO,
+        eu_demo: content_flag === ContentFlag.EU_DEMO,
+        eu_real: content_flag === ContentFlag.EU_REAL,
```

Comment:
```suggestion
        is_low_risk_cr_non_eu: content_flag === 'low_risk_cr_non_eu',
        is_low_risk_cr_eu: content_flag === 'low_risk_cr_eu',
        is_high_risk_cr: content_flag === 'high_risk_cr',
        is_cr_demo: content_flag === 'cr_demo',
        is_eu_demo: content_flag === 'eu_demo',
        is_eu_real: content_flag === 'eu_real',
```
@sergei-deriv The `content_flag` in the `traders_hub` store is already typesafe, There is no need to use that constant thing :(